docs/IE_DG/Extensibility_DG/Custom_ONNX_Ops.md

```diff
@@ -0,0 +1,59 @@
+# Custom ONNX operators {#openvino_docs_IE_DG_Extensibility_DG_Custom_ONNX_Ops}
+
+ONNX importer provides mechanism to register custom ONNX operators based on predefined or user defined nGraph operations.
+The function responsible for registering a new operator is called `ngraph::onnx_import::register_operator` and is defined in `onnx_import/onnx_utils.hpp`.
+
+## Registering custom ONNX operator based on predefined nGraph operations
+
+As an example, let's register 'CustomRelu' ONNX operator in a domain called 'com.example'.
+CustomRelu is defined as follows:
+```
+x >= 0 => f(x) = x * alpha
+x < 0  => f(x) = x * beta
+```
+where alpha, beta are float constants.
+
+Let's start with including headers:
+@snippet onnx_custom_op/main.cpp onnx_custom_op:headers
+
+Next we define a node with op_type 'CustomRelu', domain 'com.example', one input, one output and two float attributes 'alpha' and 'beta'.
```

Comment:
2. Define a node with  op_type 'CustomRelu', domain 'com.example', one input, one output and two float attributes 'alpha' and 'beta'.